roles/cinder-common/templates/etc/cinder/policy.json

```diff
@@ -11,17 +11,26 @@
     "volume:get": "",
     "volume:get_all": "",
     "volume:get_volume_metadata": "",
+    "volume:delete_volume_metadata": "rule:admin_or_owner",
+    "volume:update_volume_metadata": "rule:admin_or_owner",
     "volume:get_volume_admin_metadata": "rule:admin_api",
+    "_COMMENT": "replaced in liberty",
```

Comment:
The following line is deprecated as of Liberty. Left it in here for backwards compatibility, but it will be removed in the final PR.